### PR TITLE
merge with main

### PR DESCRIPTION
**Pull request by commando839**

### CODE REVIEW

File: Conditional Expressions/Checking_If_Age_Greater_Than_18.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+a = int(input ("Enter your age: "))
+
+if(a>18):
```
Comment (Hammad1007) [inline Conditional Expressions/Checking_If_Age_Greater_Than_18.py:3]: Can we work without braces?

File: Conditional Expressions/Detect_Spam_Comment.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+a = ["make a lot of money", "buy now", "subscribe this", "click this"]
+b = input("Enter the text here to check: ")
+
+if ("make a lot of money" in b):
```
Comment (Hammad1007) [inline Conditional Expressions/Detect_Spam_Comment.py:4]: Can you kindly explain this code to me?

File: Conditional Expressions/Grade_Calculation.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+a = int(input("Enter the marks: "))
+
+if (90<a<100):
```
Comment (Hammad1007) [inline Conditional Expressions/Grade_Calculation.py:3]: it is better to use spaces like in the following:
if 90 < a < 100

Comment (Hammad1007) [inline Conditional Expressions/Grade_Calculation.py:3]: Also let me know if the following line gives an error:
if 90 > a and a < 100:

File: Conditional Expressions/Greatest_Of_4_Numbers.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+a = input ("Enter the first number: ")
```
Comment (Hammad1007) [inline Conditional Expressions/Greatest_Of_4_Numbers.py:1]: Kindly reduce the number of lines of code while coding. Use best practices

File: Conditional Expressions/Greatest_Of_4_Numbers.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+a = input ("Enter the first number: ")
+a = int (a)
+b = input ("Enter the second number: ")
+b = int (b)
+c = input ("Enter the third number: ") 
+c = int (c)
+d = input ("Enter the fourth number: ")
+d = int (d)
+
+if(a>d):
```
Comment (Hammad1007) [inline Conditional Expressions/Greatest_Of_4_Numbers.py:10]: Can you compare all the 4 variables altogether?

File: Data Types & Variables/Loops in Python [hunk 6]
```diff
@@ -0,0 +1 @@
+
```
Comment (Hammad1007) [inline Data Types & Variables/Loops in Python:1]: Remove this line or file whatever it is

**Review by Hammad1007 — COMMENTED** [review]
Correct these errors and then start a PR again

File: Conditional Expressions/Checking_If_Age_Greater_Than_18.py (hunk 1 above)
Comment (commando839) [reply Conditional Expressions/Checking_If_Age_Greater_Than_18.py:3]: Yes

File: Conditional Expressions/Grade_Calculation.py (hunk 3 above)
Comment (commando839) [reply Conditional Expressions/Grade_Calculation.py:3]: Ok sure

File: Conditional Expressions/Greatest_Of_4_Numbers.py (hunk 5 above)
Comment (commando839) [reply Conditional Expressions/Greatest_Of_4_Numbers.py:10]: Nope the question required it that way

**Review by Hammad1007 — APPROVED** [review]
Done

**Pull request merged**